docs/excel2xml-module.md

```diff
@@ -196,7 +196,7 @@ here](./file-formats/xml-data-file.md#dsp-base-resources-and-base-properties-to-
 #### Resource ID
 
 Special care is needed when the ID of a resource is created. Every resource must have an ID that is unique in the file,
-and it must meet the constraints of xsd:ID. You can simply achieve this if you use the method `make_xsd_id_compatible()`.
+and it must meet the constraints of xsd:ID. You can simply achieve this if you use the function `make_xsd_id_compatible()`.
```

Comment:
You should mention here that it also adds a UUID based on the timestamp which cannot be reconstructed later.

Reply:
May I delegate this work to you, in https://linear.app/dasch/issue/DEV-4026/featexcel2xml-new-function-make-xsd-id-compatible-without-uuids?